documentation/testnet/getting_started/02_deploy_and_execute.md

```diff
@@ -182,7 +182,7 @@ Transfer credits with a `credits.aleo` program execution.
 
 ##### Arguments:
 
-- `input_record` - The record used to craft the transfer
+- `input-record` - The record used to craft the transfer
```

Comment:
```suggestion
- `input_record` - The record used to craft the transfer
```
I believe these are underscores https://github.com/AleoHQ/snarkOS/blob/a7cf1b4cb4dab5e66dee6f7b68ab2981af0c2690/cli/src/commands/developer/transfer_private.rs#L31

Can you the commands and double check?

Reply:
Yes for sure, I tested the commands earlier with snarkOS on the testnet3 branch, see screenshot [here](https://github.com/AleoHQ/welcome/pull/221#discussion_r1242658658). I was only able to execute the commands with "-" not "_" on my end.

Reply:
great thanks for checking